mmf/models/transformers/base.py

```diff
@@ -103,25 +103,13 @@ def build(self):
 
     def get_optimizer_parameters(self, config):
         lr = config.optimizer.params.lr
-
+        backbone_param_set = set()
```

Comment:
Let's name this `trunk_params_set`.

Reply:
Thanks for your reviewing. I will change my code according to ur suggestion.